src/Routing/CachingContentfulSlugMatcher.php

```diff
@@ -0,0 +1,163 @@
+<?php
```

Comment:
But this is another style of class as we talked about?

Reply:
A decorator did not work?

Reply:
It did. But it would have beed more overhead. The need to implement the interfaces again. And the bundle would have to declare two routers just to use the caching one.